python/private/py_wheel.bzl

```diff
@@ -352,6 +414,33 @@ def _py_wheel_impl(ctx):
         ),
     ]
 
+def _inputs_to_package(actions, workspace_name, deps, links_subdir, linker):
+    filtered = []
+    for dep in deps:
+        files = dep[PyInfo].transitive_sources
+        imports = [imports.replace("\\", "/") for imports in dep[PyInfo].imports.to_list()]
```

Comment:
Where is the \\ coming from? Windows? Please add a comment to that effect.

Reply:
In more than 20 years in programming, I never saw a case where a backslash in a path meant something else other than dealing with Windows, and because it's so obvious, I won't add a comment.